hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/HASecurityUtils.java

```diff
@@ -93,6 +93,11 @@ public static void initializeSecurity(SCMStorageConfig scmStorageConfig,
     CertificateClient certClient =
         new SCMCertificateClient(new SecurityConfig(conf));
     InitResponse response = certClient.init();
+    if (response.equals(CertificateClient.InitResponse.REINIT)) {
```

Comment:
This is for SCM.  We support SCM reinit?

Reply:
Yes you are right, I have removed this code, and this way SCM will still fail if the certificate is expired, but that is still expected, and it is expected by default just after 5 years.